.eslintrc

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "loopback",
   "rules": {
-    "max-len": ["error", 125, 4, {
+    "max-len": ["error", 140, 4, {
```

Comment:
any special reason to increase the length?

Reply:
when I run npm test, giving error saying needed 140. so i changed  it. 
(how to run github actions running for CI, Did i miss some thing )

Reply:
Sorry about the confusion. The GitHub Actions comment is for future reference. :) 
Currently I'm verifying locally, but ideally we need to get GitHub Actions working for CI. But that's out of scope for this PR.